### ISSUE TITLE
Consider signing install script or installer program

### ISSUE THREAD

**Issue by waryhermit** [dogi/ole--vagrant-vi#19]
People should be reluctant to run unsigned scripts on their system. For now we can just point them to the script and them them decide if it is safe for themselves, but in the future it would be nice to have it signed so people don't have to change their execution policy to something unsafe to install the software.

**Comment by dogi** [comment]
@waryhermit are you proposing to add security when downloading/installing?

**Comment by waryhermit** [comment]
@dogi Yes, I didn't phrase it well, but windows has the execution of powershell scripts disabled by default because of security issues. If we signed it they could keep the execution policy restricted to signed scripts only and keep their system more secure. Not a pressing issue for sure, but I wanted to make an issue to keep it in the back of our minds.

**Comment by dogi** [comment]
@waryhermit thanks I understand now

what are the next steps here?

**Comment by empeje** [comment]
Hi @dogi and @waryhermit 

Sorry to jump in, I see this link https://www.darkoperator.com/blog/2013/3/5/powershell-basics-execution-policy-part-1.html helpful, and since I lost all my ole vagrant file I will try to assign this to myself.

**Comment by waryhermit** [comment]
@dogi @mappuji I don't think its important enough for anyone to work on right now. Normally we can bypass their execution policy because we have them paste it into a command prompt - only trouble is when I have to help someone figure out why it didn't work that their execution policy makes it hard. Not sure if I am clear or not.
